dashboard/cron.py

```diff
@@ -303,11 +303,8 @@ def update_with_bq_access(self):
             logger.debug(resource_access_df)
 
             # Because we're pulling all the data down into one query we need to manipulate it a little bit
-            # Make a copy of the access dataframe
-            resource_df = resource_access_df.copy(deep=True)
-            # Drop out the columns user and access time from resource data frame
-            resource_df = resource_df.drop(["user_id", "access_time"], axis=1)
-            # Drop out the duplicates
+            # This creates a new resource df of the resource_access with specific columns
+            resource_df = resource_access_df.filter(["resource_id", "resource_type", "name"], axis=1)
             resource_df = resource_df.drop_duplicates(["resource_id"])
```

Comment:
I feel bad about leaving you in the lurch with a conflict.  I was being encouraged to merge my PR ASAP.

Here's what the conflict involves:

```
<<<<<<< issue_1017 -- Incoming Change
            # Because we're pulling all the data down into one query we need to manipulate it a little bit
            # This creates a new resource df of the resource_access with specific columns
            resource_df = resource_access_df.filter(["resource_id", "resource_type", "name"], axis=1)
            resource_df = resource_df.drop_duplicates(["resource_id"])
=======
            # Make resource data from resource_access data
            resource_df = resource_access_df.copy(deep=True)
            resource_df = resource_df.drop(
                columns=['user_id', 'access_time', 'course_id'])
            resource_df = resource_df.drop_duplicates('resource_id')

            # set resource_id as index for pangres.upsert
            resource_df: pd.DataFrame = resource_df.set_index('resource_id')
>>>>>>> master -- Current Change
```

Where the upper part is your code and the lower one contains my change.  I think it can be resolved like this:

```suggestion
            # Make resource data from resource_access data
            resource_df = resource_access_df.filter(
                columns=['resource_id', 'resource_type', 'name'])
            resource_df = resource_df.drop_duplicates('resource_id')
```

It's mostly a matter of choosing your changes instead of mine.

Reply:
I forgot to say:  The `resource_df.set_index()` part of the conflict threw me off.  At first I thought you had dropped that from the code because it wasn't necessary.  However, I see that you still have it, but I had reformatted it in my code.  It was different enough to confuse the conflict detection in the diff.  (Also the conflict editor was WAY off on line numbers.  It said these changes were around line 366, when they're actually around 305.)

Reply:
Yeah this code didn't have much in it, resolving this shouldn't take long, probably even less time than replying to these comments! ;)

And yeah, all of this could use a good refactoring to make it easier to manage, though we've been trying to avoid too many ETL changes pending future backend changes (like UDP).

Reply:
Okay, just resolving it like this.